### ISSUE TITLE
Alarm stopped showing state

### ISSUE THREAD

**Issue by jirihelmich** [Pigotka/ha-cc-jablotron-cloud#29]
During the night, the Alarm became Unavailable and then Unknown. Therefore it stopped propagating through HomeKit Bridge into HomeKit. Arming and disarming seems to work, but the state is Unknown.

Here's some images and logs related:
```
2024-11-07 09:08:09.956 DEBUG (MainThread) [custom_components.jablotron_cloud] Finished fetching Jablotron Cloud data in 2.366 seconds (success: True)
2024-11-07 09:08:09.956 DEBUG (MainThread) [custom_components.jablotron_cloud.alarm_control_panel] Updating section state: {'cloud-component-id': 'SEC-*****', 'state': 'DISARM'}
2024-11-07 09:08:09.957 WARNING (MainThread) [homeassistant.components.alarm_control_panel] Entity alarm_control_panel.alarm_raskova_upstairs (<class 'custom_components.jablotron_cloud.alarm_control_panel.JablotronAlarmControlPanel'>) is setting state directly which will stop working in HA Core 2025.11. Entities should implement the 'alarm_state' property and return its state using the AlarmControlPanelState enum, please report it to the custom integration author.
```

```
2024-11-07 08:07:14.619 ERROR (MainThread) [pyhap.characteristic] SecuritySystemCurrentState: value=0 is an invalid value.
2024-11-07 08:07:14.619 ERROR (MainThread) [pyhap.characteristic] SecuritySystemTargetState: value=0 is an invalid value.
2024-11-07 08:07:14.619 ERROR (MainThread) [homeassistant.components.homekit] Failed to create a HomeKit accessory for alarm_control_panel.alarm_raskova_upstairs
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/homekit/__init__.py", line 758, in add_bridge_accessory
    acc = get_accessory(self.hass, self.driver, state, aid, conf)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/accessories.py", line 280, in get_accessory
    return TYPES[a_type](hass, driver, name, state.entity_id, aid, config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 141, in __init__
    self.async_update_state(state)
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 160, in async_update_state
    hass_state = AlarmControlPanelState(new_state.state)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 'unknown' is not a valid AlarmControlPanelState
2024-11-07 08:07:14.623 ERROR (MainThread) [pyhap.characteristic] SecuritySystemCurrentState: value=0 is an invalid value.
2024-11-07 08:07:14.623 ERROR (MainThread) [pyhap.characteristic] SecuritySystemTargetState: value=0 is an invalid value.
2024-11-07 08:07:14.624 ERROR (MainThread) [homeassistant.components.homekit] Failed to create a HomeKit accessory for alarm_control_panel.alarm_raskova_garage
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/homekit/__init__.py", line 758, in add_bridge_accessory
    acc = get_accessory(self.hass, self.driver, state, aid, conf)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/accessories.py", line 280, in get_accessory
    return TYPES[a_type](hass, driver, name, state.entity_id, aid, config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 141, in __init__
    self.async_update_state(state)
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 160, in async_update_state
    hass_state = AlarmControlPanelState(new_state.state)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 'unknown' is not a valid AlarmControlPanelState
2024-11-07 08:07:14.631 ERROR (MainThread) [pyhap.characteristic] SecuritySystemCurrentState: value=0 is an invalid value.
2024-11-07 08:07:14.631 ERROR (MainThread) [pyhap.characteristic] SecuritySystemTargetState: value=0 is an invalid value.
2024-11-07 08:07:14.631 ERROR (MainThread) [homeassistant.components.homekit] Failed to create a HomeKit accessory for alarm_control_panel.alarm_raskova_downstairs
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/homekit/__init__.py", line 758, in add_bridge_accessory
    acc = get_accessory(self.hass, self.driver, state, aid, conf)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/accessories.py", line 280, in get_accessory
    return TYPES[a_type](hass, driver, name, state.entity_id, aid, config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 141, in __init__
    self.async_update_state(state)
  File "/usr/src/homeassistant/homeassistant/components/homekit/type_security_systems.py", line 160, in async_update_state
    hass_state = AlarmControlPanelState(new_state.state)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 'unknown' is not a valid AlarmControlPanelState
```

<img width="618" alt="image" src="https://github.com/user-attachments/assets/697a23db-7c47-4e08-abd6-cb650bcd589a">

**Comment by DaRockwilder187** [comment]
Hello, 
get the same since update of Homeassistant. Opened issue parallel at the same time:
https://github.com/Pigotka/ha-cc-jablotron-cloud/issues/30#issue-2640251470

**Comment by Pigotka** [comment]
Did that also happen after updating to the latest HA?

**Comment by karelkryda** [comment]
It is interesting that according to the [documentation](https://www.home-assistant.io/integrations/alarm_control_panel#the-state-of-an-alarm-panel-entity), the `unknown` status is valid.

However, according to the developer's [documentation](https://developers.home-assistant.io/docs/core/entity/alarm-control-panel/#states), it is not.

**Comment by karelkryda** [comment]
Just FYI @jirihelmich, I found related [issue](https://github.com/home-assistant/core/issues/129867) in hassio core repo

**Comment by DaRockwilder187** [comment]
> Did that also happen after updating to the latest HA?

seems so, did the update last night and recognized this issue some time (guess 2h) after it was done.

**Comment by jirihelmich** [comment]
True, I saw an update of HA last evening.

**Comment by Pigotka** [comment]
Resolved in HA 24.11.2 so you can update.
